Xamarin.Forms.Core/MergedStyle.cs

```diff
@@ -62,8 +62,8 @@ public IList<string> StyleClass
 				if (_styleClass != null) {
 					_classStyleProperties = new List<BindableProperty> ();
 					foreach (var styleClass in _styleClass) {
-						var classStyleProperty = BindableProperty.Create ("ClassStyle", typeof(IList<Style>), typeof(VisualElement), default(IList<Style>),
-							propertyChanged: (bindable, oldvalue, newvalue) => ((VisualElement)bindable)._mergedStyle.OnClassStyleChanged());
```

Comment:
I just removed the cast because AFAICT _mergedStyle will always just be `this`

If that's not true then I think MergedStyle should be added to `IStyleSelectable`